ReactNativeDeferredTemplate/.flowconfig

```diff
@@ -0,0 +1,58 @@
+[ignore]
+
+# We fork some components by platform.
```

Comment:
Are these files meant to be checked in or a part of `.gitignore`?

Reply:
There are meant to be checked in - they are part of ReactNativeTemplate also. They appear in the project generated with react native cli (or at least they did they last time I ran the react native cli).